docs/client_reference.rst

```diff
@@ -179,6 +179,11 @@ The client session supports the context manager protocol for self closing.
 
       Get proxy credentials from ``~/.netrc`` file if present.
 
+      Get HTTP Basic Auth credentials from :file:`~/.netrc` file if present.
+
+      If :envvar:`NETRC` environment variable is set, read from file specified
```

Comment:
It looks like https://aiohttp--7131.org.readthedocs.build/en/7131/glossary.html needs an entry for the env var to be linkable. It's currently unlinked: https://aiohttp--7131.org.readthedocs.build/en/7131/client_reference.html#aiohttp.ClientSession.

Reply:
I've added an `..envvar` directive there, but it is now at the bottom of the page rather than in line. I can instead switch the `:envvar:` here to `:term:` and just put `NETRC` into the glossary if you would prefer